configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5s.sh

```diff
@@ -0,0 +1,15 @@
+ #!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DECODE="decodebin force-sw-decoders=1" #decodebin|vaapidecodebin
+DEVICE="CPU" #GPU|CPU|MULTI:GPU,CPU
+PRE_PROCESS="" #""|pre-process-backend=vaapi-surface-sharing|pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 
+AGGREGATE="gvametaaggregate name=aggregate !" # Aggregate function at the end of the pipeline ex. "" | gvametaaggregate name=aggregate
+OUTPUTFORMAT="! fpsdisplaysink video-sink=fakesink sync=true --verbose" # Output format for the pipeline  "! fpsdisplaysink video-sink=fakesink sync=true --verbose" | (render_mode)"! videoconvert ! video/x-raw,format=I420 ! gvawatermark ! videoconvert ! fpsdisplaysink video-sink=ximagesink sync=true --verbose"
+
+echo "run pipeline"
```

Comment:
could you add what kind of pipeline is running here? eg. `run yolov5s detection only pipeline`